packages/server/src/api/statistic-definitions.js

```diff
@@ -9,6 +9,18 @@ const {computeRepresentativeRuns} = require('@lhci/utils/src/representative-runs
 
 /** @typedef {(lhrs: Array<LH.Result>) => ({value: number})} StatisticFn */
 
+/**
+ * @return {StatisticFn}
+ */
+function metaLighthouseVersion() {
+  return lhrs => {
+    const version = lhrs[0].lighthouseVersion || '';
+    const [_, major = '0', minor = '0', patch = '0'] = version.match(/^(\d+)\.(\d+)\.(\d+)/) || [];
```

Comment:
do you care about the alpha/beta tag?

Reply:
not really. we'll never publish a version of lhci that's stuck on an alpha or beta tag and in the event that someone uploads data manually to lhci server using `lighthouse@next` I'm ok with the idea that they won't get warnings between alpha/beta/final versions

this behavior is also captured by the tests for this file so unlikely to accidentally change it either